ansible/roles/ocp4-workload-tektoncd-foundations/tasks/workload.yml

```diff
@@ -0,0 +1,80 @@
+---
+# Implement your Workload deployment tasks here
+
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+- name: Install tekton CLI (tkn)
+  become: yes
+  unarchive:
+    src: "{{ _tekton_cli_url }}"
+    remote_src: yes
+    dest: /usr/bin
```

Comment:
Consider putting these in `/usr/local/bin|sbin` as a more appropriate location since these are programs not controlled by package manager. 

Same for `yq` and `kn`

Reply:
@stencell Added to `/usr/local/bin` as suggested